### PR TITLE
Two of three missing sensor types implemented.

### PR DESCRIPTION
**Pull request by Yogibaer75**
No brightness sensor was available on the used device.
Originally this extension for the etherbox check was inside my own collection since 2014 :)

Thank you for your interest in contributing to Checkmk!
Unfortunately, due to our current work load, we only consider pure bug fixes as stated in our [Readme](https://github.com/tribe29/checkmk#want-to-contribute).
This means any new pull request that is not a pure bug fix will be closed.
Instead of creating a PR, please consider sharing new check plugins, agent plugins, special agents or notification plugins via the [Checkmk Exchange](https://exchange.checkmk.com/).

## General information

Please give a brief summary of the affected device, software or appliance.
Keep in mind that we are experts in monitoring, but we cannot be experts on all supported devices.
A little context will help us assess your proposed change.

## Bug reports

Please include:

+ Your operating system name and version
+ Any details about your local setup that might be helpful in troubleshooting
+ Detailed steps to reproduce the bug
+ An agent output or SNMP walk
+ The ID of a submitted crash report for reference (if applicable)

## Proposed changes

Sometimes it is hard for us to assess the quality of a fix.
While it may work for you, it is our job to ensure that it works for everybody.
These are some ways to help us:

+ What is the expected behavior?
+ What is the observed behavior?
+ If it's not obvious from the above: In what way does your patch change the current behavior?
+ Consider writing a unit test that would have failed without your fix.
+ Is this a new problem? What made you submit this PR (new firmware, new device, changed device behavior)?

### CODE REVIEW

**Comment by Yogibaer75** [comment]
It's "very nice" to have the output "it is not allowed to add new legacy checks" if you try to fix a 10 year old bug.
If this complete check is already ported to the new check type it would also be possible to fix the new check. :(

**Comment by kain88-de** [comment]
I can see if I have some time to migrate the plugin. Can you migrate your fix afterwards?

File: checks/etherbox [hunk 1]
```diff
@@ -220,3 +220,68 @@ check_info["etherbox.smoke"] = {
...
+def check_etherbox_voltage(item, no_params, info):
+    try:
+        name, value = etherbox_get_sensor(item, "5", info)
+    except Exception as error:
+        return 3, str(error)
+
+    state = 0
+    perfdata = [("voltage", value)]
+    extra_info = " - Status: OK"
+    if value != "0":
```
Comment (kain88-de) [inline checks/etherbox:244]: I'm currently looking to implement those check in a migrated version of this check. Should there really be directly an alarm if voltage is unequal to zero? I would assume that this is better with configurable levels.

Comment (Yogibaer75) [reply checks/etherbox:244]: I must take a look at the old plugin then i can answer this :)

Comment (Yogibaer75) [reply checks/etherbox:244]: I used the same procedure as for smoke :)
The problem is that also for smoke there should be configurable values.
Inside the docs you find as smoke values something like "lower thresholds 0 / 0 , upper thresholds 90 / 110".
Now the problem, for the voltage sensor there is no documentation available.

Comment (kain88-de) [reply checks/etherbox:244]: So we make this configurable and use 0 as the default, for smoke as well. Behavior is not changed but can be adopted later.

**Comment by kain88-de** [comment]
Superseded by #499

**Pull request closed without merging**